extensions/projection/README.md

```diff
@@ -0,0 +1,93 @@
+# Projection Extension Specification (`proj`)
+
+**Extension [Maturity Classification](../README.md#extension-maturity): Proposal**
+
+This document explains the fields of the STAC Projection (`proj`) Extension to a STAC Item. Here `proj` is short
+for "projection", and not a reference to the use of the PROJ format for the CRS attribute.
+
+The field names defined herein should be added as fields in the Item's Properties object. 
+
+When specified on an Item, the values are assumed to apply to all Assets in that Item.  For example, an Item may have several related Assets each representing a band for the Item, all of which are in the same native CRS, e.g., a UTM Zone.  However, there may also be assets intended for display, like a preview image or thumbnail, that have been reprojected to a different CRS, e.g., Web Mercator, to better accommodate that use case.  This case of differing projections per Asset is not currently handled by this extension.
+
+## Item Properties fields
+
+| Field Name       | Type                     | Description |
+| ---------------- | ------------------------ | ----------- |
+| proj:epsg        | integer\|null   | **Required** [EPSG code](http://www.epsg-registry.org/) of the datasource |
+| proj:crs         | string \|null   | Recommended. PROJ string representing the Coordinate Reference System (CRS) that the `proj:geometry` and `proj:bbox` fields represent |
```

Comment:
Using PROJ is discouraged: https://twitter.com/EvenRouault/status/1189515652958212096
We should switch this field to use WKT(2) and/or PROJJSON.

Reply:
I don't have a strong opinion on this.  The intention of this field was as a fallback when there's no EPSG code, e.g., MODIS Sinusoidal. I doubt it will be used in practice very frequently, and almost certainly not for search. It's primarily as metadata to avoid needing to retrieve the imagery bits to get the CRS.